### DISCUSSION TITLE
Add og:image tag to Developer Portal metadata

### DISCUSSION THREAD

**Discussion by hansukyang** [Azure/api-management-developer-portal#2589] [Feature request]
It would be great if there is a way to add og metadata property (https://ogp.me/) for sharing the developer portal via social media, at least for the home page.

Example from the link:

```
<title>The Rock (1996)</title>
<meta property="og:title" content="The Rock" />
<meta property="og:type" content="video.movie" />
<meta property="og:url" content="https://www.imdb.com/title/tt0117500/" />
<meta property="og:image" content="https://ia.media-imdb.com/images/rock.jpg" />
...
</head>
```

The main benefit would be to control the image that gets shown when the developer portal page is shared on LinkedIn, Twitter, Facebook etc.

**Comment by ghost** [comment]
@hansukyang, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@hansukyang, thank you for requesting this feature.

**Comment by sid6mathur** [comment]
+1 for this HTML hygiene feature for the 2020s :)
